li-apache-kafka-clients/src/main/java/com/linkedin/kafka/clients/consumer/LiKafkaConsumerConfig.java

```diff
@@ -36,6 +38,7 @@
   public static final String ENABLE_AUTO_COMMIT_CONFIG = ConsumerConfig.ENABLE_AUTO_COMMIT_CONFIG;
   public static final String AUTO_COMMIT_INTERVAL_MS_CONFIG = ConsumerConfig.AUTO_COMMIT_INTERVAL_MS_CONFIG;
   public static final String AUTO_OFFSET_RESET_CONFIG = ConsumerConfig.AUTO_OFFSET_RESET_CONFIG;
+  public static final String MAX_POLL_RECORDS_CONFIG = ConsumerConfig.MAX_POLL_RECORDS_CONFIG;
```

Comment:
dont we implicitely inherit everything from vanilla consumer anyway? why do we need to call this out here ?

Reply:
LiKafkaConsumerConfig currently does not inherit ConsumerConfig and needs to define all configs used. Otherwise, getters in AbstractConfig will throw a ConfigException for unknown ones. Same story for LiKafkaProducerConfig.

I don't know why it was written in this way, though. Do you remember?